styles/LexArcana.css

```diff
@@ -0,0 +1,236 @@
+*
+{
+    margin: 0;
+    padding: 0;
+}
+
+table td
+, .mce-content-body td
+{
+    margin: 5px;
+    padding: 5px;
+}
+
+.system-LexArcana input
+, .system-LexArcana .sidebar-tab button
+, .system-LexArcana .sidebar-tab .window-content
+, .system-LexArcana .sidebar-tab a
+, .system-LexArcana .sidebar-tab .encounters
+, .system-LexArcana .sidebar-tab .directory-list .folder
+, .system-LexArcana .sidebar-tab .chat-message
+{
+    font-family: Herculanum;
+    font-size: 14px;
+    font-weight: 700;
+    text-shadow: 0px 1px 0px rgba(255,255,255,.3), 0px -1px 0px rgba(0,0,0,.7);
+}
+
+.system-LexArcana .window-content
+{
+    background-image:url("../styles/images/marble-black.png");
+    background-repeat: repeat;
+    background-size: 512px 512px;
+    color: #ffffff;
+}
+
+.system-LexArcana .window-content .sheet-header
+{
+    max-height: 64px;
+}
+
+.system-LexArcana .window-content .sheet-header ul
+{
+    list-style: none;
+}
+
+.system-LexArcana .window-app h3
+, .system-LexArcana .window-app h4
+{
+    font-size: 18px;
+    font-weight: bold;
+    border-bottom: 1px solid #d7000055;
+}
+
+/**********************************************************
+ * Character sheets
+ **********************************************************/
+
+.system-LexArcana .sheet-navigation
+{
+    color: #ffffffaa;
+    background-color: #ffffff55;
+    text-align: center;
+    line-height: 32px;
+    max-height: 32px;
+}
+
+.system-LexArcana .abilities li
+{
+    list-style: none;
+    margin-top: 15px;
+    margin-bottom: auto;
```

Comment:
I know why you used such a ```margin-bottom``` but I believe you could have the same result using ```align-items: flex-start``` since you are inside a flex block. See the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container) for more information

Reply:
I tried, I had to add some lines more and it complexified a lot the whole css making it even uglier and difficult to read. So I'll stick with margin-bottom, simple and does the job as intended